ompi/mca/pml/pml.h

```diff
@@ -499,6 +514,9 @@ struct mca_pml_base_module_1_0_1_t {
     /* downcalls from MPI to PML */
     mca_pml_base_module_add_comm_fn_t     pml_add_comm;
     mca_pml_base_module_del_comm_fn_t     pml_del_comm;
+#if OPAL_ENABLE_FT_MPI
+    mca_pml_base_module_revoke_comm_fn_t  pml_revoke_comm;
```

Comment:
Adds a field to the module ABI in a conditional compilation. Make non-conditional?

Reply:
+1 :)